src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/X509Pal.PublicKey.cs

```diff
@@ -32,6 +32,11 @@ public AsymmetricAlgorithm DecodePublicKey(Oid oid, byte[] encodedKeyValue, byte
                 return DecodeECDsaPublicKey((CertificatePal)certificatePal);
             }
 
+            return DecodePublicKey(oid, encodedKeyValue, encodedParameters);
+        }
+
+        public AsymmetricAlgorithm DecodePublicKey(Oid oid, byte[] encodedKeyValue, byte[] encodedParameters)
```

Comment:
I did it this way assuming the existing one that depends on the cert could go away (assuming all of the information are in the parameters)